docs/serverless-lambda.asciidoc

```diff
@@ -1,69 +1,29 @@
 [[lambda-support]]
 === Monitoring AWS Lambda Python Functions
-:layer-section-type: extension-only
+:layer-section-type: with-agent
 :apm-aws-repo-dir: ./lambda
 
-Incorporating Elastic APM into your AWS Lambda functions is easy!
-Follow the steps below to setup Elastic APM for your AWS Lambda Python functions.
+The Python APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
 
 [float]
 ==== Prerequisites
 
 You need an APM Server to send APM data to. Follow the {apm-guide-ref}/apm-quick-start.html[APM Quick start] if you have not set one up yet. For the best-possible performance, we recommend setting up APM on {ecloud} in the same AWS region as your AWS Lambda functions.
 
 [float]
-==== Step 1: Set up the {apm-lambda-ext}
+==== Step 1: Select the AWS Region and Architecture
 
 include::{apm-aws-lambda-root}/docs/lambda-selector/lambda-attributes-selector.asciidoc[]
 
-Add the {apm-lambda-ref}/aws-lambda-arch.html[{apm-lambda-ext}] as an https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html[AWS Lambda Layer] to your AWS Lambda function.
-
-include::{apm-aws-lambda-root}/docs/lambda-selector/extension-arn-replacement.asciidoc[]
-include::{apm-aws-lambda-root}/docs/add-extension/add-extension-layer-widget.asciidoc[]
-
 [float]
-==== Step 2: Set up the APM Python Agent
-
-You need to add `elastic-apm` as a dependency for your python function.
-Depending on your deployment strategy, this could be as easy as adding
-`elastic-apm` to your `requirements.txt` file, or installing it in the directory
-you plan to deploy using pip:
-
-[source,bash]
-----
-$ pip install -t <target_dir> elastic-apm
-----
-
-Note: Please use the latest version of the APM Python agent. A performance
-issue was introduced in version 6.9.0 of the agent, and fixed in version 6.14.0.
-
-Once the library is included as a dependency in your function, you must
-import the `capture_serverless` decorator and apply it to your handler:
-
-[source,python]
-----
-from elasticapm import capture_serverless
-
-@capture_serverless
-def handler(event, context):
-    return {"statusCode": r.status_code, "body": "Success!"}
-----
-
-Note: any database or connection pool setup should go inside of your handler,
-otherwise the agent will not be able to instrument those objects and you may
-see missing spans. Example:
+==== Step 2: Add the APM Layers to your Lambda function
 
+include::{apm-aws-lambda-root}/docs/lambda-selector/extension-arn-replacement.asciidoc[]
+include::./lambda/python-arn-replacement.asciidoc[]
```

Comment:
@basepi The following file is missing in this PR: `./lambda/python-arn-replacement.asciidoc`
I think that's why the placeholder `<SELECTED_AGENT_LAYER_ARN>` is not being replaced.

You need to add that file with the following content:
```
++++
<script>
window.addEventListener("DOMContentLoaded", async () => {
  addArnGenerator('agent', 'apm-agent-python', 'arn:aws:lambda:${region}:267093732750:layer:elastic-apm-python-${version}');
  replaceAgentDockerImageParams('FROM docker.elastic.co/observability/apm-agent-python:latest AS python-agent',
                                'COPY --from=python-agent /opt/python/ /opt/python/');
});
</script>
++++
```

See the example from Node.js: https://github.com/elastic/apm-agent-nodejs/tree/main/docs/lambda